bot/exts/fun/movie.py

```diff
@@ -71,28 +77,34 @@ async def movies(self, ctx: Context, genre: str = "", amount: int = 5) -> None:
         # Capitalize genre for getting data from Enum, get random page, send help when genre don't exist.
         genre = genre.capitalize()
         try:
-            result = await self.get_movies_data(self.http_session, MovieGenres[genre].value, 1)
+            result, status = await self.get_movies_data(self.http_session, MovieGenres[genre].value, 1)
         except KeyError:
             await invoke_help_command(ctx)
             return
 
         # Check if "results" is in result. If not, throw error.
         if "results" not in result:
             err_msg = (
-                f"There is problem while making TMDB API request. Response Code: {result['status_code']}, "
-                f"{result['status_message']}."
+                f"There is problem while making TMDB API request. Response Code: {status}, "
```

Comment:
```suggestion
                f"There was a problem making the TMDB API request. Response Code: {status}, "
```
and same below, it could be nice if the repetition could be removed somehow, but you don't have to since it was already like that.

Reply:
TBH I don't really get it, why the original author had to 2 requests, with varying `page`.